### ISSUE TITLE
UNMET PEER DEPENDENCY react@16.0.0-alpha.12

### ISSUE THREAD

**Issue by Adamnator92** [airbnb/lottie-react-native#156]
```

├─┬ lottie-react-native@1.2.0 
│ ├── lottie-ios@1.5.2 
│ └─┬ react-native-safe-module@1.2.0 
│   └── dedent@0.6.0 
└── UNMET PEER DEPENDENCY react@16.0.0-alpha.12
```

npm WARN lottie-react-native@1.2.0 requires a peer of react@>=15.3.1 but none was installed.

i having a trouble during npm install `lottie-react-native`

### My Environment

| software         | version
| ---------------- | -------
| react | "16.0.0-alpha.12",
| react-native     |  "0.46.1"

**Comment by thainx** [comment]
I have a similar problem. but with react@16.0.0-alpha.13 and react-native@0.46.4
Even worse, everytime I start the app, it shows like this
![image](https://user-images.githubusercontent.com/17704842/28604078-6b80e18a-71f2-11e7-9a27-7191ca1548c9.png)

It seems that the package still use React.PropTypes (which was removed in react 16)

**Comment by jrnk** [comment]
See https://github.com/airbnb/lottie-react-native/pull/161

**Comment by gpeal** [comment]
@Adamnator92 @thainx @jrnk This will be resolved in 2.2.0 which I'll release later today :)
